Gemfile

```diff
@@ -1,27 +1,27 @@
-source 'https://rubygems.org'
+source "https://rubygems.org"
 
-ruby File.read('.ruby-version').chomp
+ruby File.read(".ruby-version").chomp
 
-gem 'bundler-audit', git: 'https://github.com/rubysec/bundler-audit', branch: 'master'
-gem 'octokit', '~> 4.3'
-gem 'rake', '~> 12.3.0'
-gem 'sinatra'
-gem 'slack-poster', '~> 2.2'
+gem "bundler-audit", git: "https://github.com/rubysec/bundler-audit", branch: "master"
```

Comment:
Why do we use this? Especially from the master branch? Seems like it does [the same or similar things as Dependabot](https://github.com/rubysec/bundler-audit)?

Reply:
Seems like it was introduced in #50 with the only context as to why being in a Trello card I can't see. 🤔

Reply:
Hmm. This is well outside this PR, but if we ever wanted to spend time to make this more standard given we already use the GitHub API for other things here, [the GitHub GraphQL API supports Security Alert info](https://developer.github.com/v4/object/repositoryvulnerabilityalert/).